### ISSUE TITLE
Incorrect TESP directory structure

### ISSUE THREAD

**Issue by Iamamun58** [pnnl/tesp#90]
I am using a VM and Ubuntu 22.04 LTS. After using the command: 
`wget --no-check-certificate https://raw.githubusercontent.com/pnnl/tesp/main/scripts/tesp.sh
chmod 755 tesp.sh
./tesp.sh <Github user name> <Github email address>`

The tesp directory is not similar as mentioned in the document. 
For me its: 

Home
├──tesp
.              ├── repository
.               ├── tenv
There is no **venv** folder in the directory. Also, `source tespEnv` not working in the terminal. 

When simulating SGIP1b.sh it is showing errors: 

`bash: cd: /home/<username>/tesp/examples/sgip1: No such file or directory`
`FileNotFoundError: [Errno 2] No such file or directory: '$TESPDIR/data/weather/aZ-Tucson_International_Ap.tmy3'`

**Comment by d3j331** [comment]
I will test for 22.04 today, I suspect that python 3.10 installed and we are using 3.8 and things didn't go as planned.

**Comment by d3j331** [comment]
This should fix your problem.  In ubuntu 22.04 has python 3.10 and we need to install 3.8 a different repo. 

sudo rm -r tesp
sudo add-apt-repository ppa:deadsnakes/ppa -y
sudo apt-get update
./tesp.sh <user> <email>

**Comment by trevorhardy** [comment]
Following those instructions above and not building the binaries did allow me to successfully install on 22.04:

```
++++++++++++++  Compiling and Installing TESP software is complete!  ++++++++++++++

FNCS, installed

HELICS, 3.3.0 (2022-09-15)

HELICS Java, 3.3.0 (2022-09-15)


GridLAB-D 5.0.0-19269 (e672c529:develop) 64-bit LINUX RELEASE

EnergyPlus, Version 9.3.0-fd4546e21b (No OpenGL)

NS-3, installed

Ipopt 3.13.2 (x86_64-pc-linux-gnu), ASL(20190605)


++++++++++++++  TESP versions has been installed! That's all folks!  ++++++++++++++

vmuser@vmuser-Parallels-Virtual-Platform:~$ . tespEnv
(TESP) vmuser@vmuser-Parallels-Virtual-Platform:~$
```

**Comment by d3j331** [comment]
Fix the issue, v1.2.2
